lib/lambda_ethereum_consensus/state_transition/cache.ex

```diff
@@ -0,0 +1,91 @@
+defmodule LambdaEthereumConsensus.StateTransition.Cache do
+  @moduledoc """
+  Caches expensive function calls.
+  """
+
+  alias LambdaEthereumConsensus.StateTransition.{Accessors, Misc}
+
+  @spec init_cache_tables() :: :ok
+  def init_cache_tables do
+    [:total_active_balance, :beacon_proposer_index, :beacon_committee, :active_validator_count]
+    |> Enum.each(fn table ->
+      :ets.new(table, [:set, :public, :named_table])
```

Comment:
how are values cleaned up? Does this ets grow indefinitely?

Reply:
Yeah. I'm leaving the clean up for a future PR. Basically, we should leave only current + previous epoch data.